src/popup/components/TransactionTagList.vue

```diff
@@ -50,11 +50,18 @@ export default defineComponent({
     dense: Boolean,
   },
   setup(props, { root }) {
+    const getTxType = computed(() => root.$store.getters.getTxType);
```

Comment:
I'm not 100% sure, but this can be removed. You are passing the `tx` to the `useTransactionTx` composable and there is `txType` taken from the `tx`.

Reply:
There are two transactions within new type of tx we are trying to support `innerTx` and `externalTx`. We are trying to get labels for the both of them. That is why we are getting type for both of them.